examples/test1.rs

```diff
@@ -30,16 +30,18 @@ fn main() -> Result<(), failure::Error> {
         .setup(|ssh, _| ssh.cmd("sudo apt update").map(|(_, _)| ()));
     b.add("india", m).unwrap();
 
-    b.wait_limit(time::Duration::from_secs(60));
-    let ts = b.spawn()?;
-    let vms = ts.get_machines()?;
-    let log = ts.logger();
+    let mut l: tsunami::providers::aws::AWSLauncher<_> = Default::default();
+    l.with_credentials(|| Ok(DefaultCredentialsProvider::new()?));
```

Comment:
Should this still call `wait_limit` like before?

Reply:
yes, except `wait_limit` is called `timeout` now.